src/components/Editor/MediaHandler.vue

```diff
@@ -6,7 +6,7 @@
 <template>
 	<div class="editor editor-media-handler"
 		data-text-el="editor-media-handler"
-		:class="{ draggedOver }"
+		:class="{ draggedOver, 'editor-media-handler-mobile': isMobile }"
```

Comment:
Nitpick for the naming to fit https://getbem.com/naming/ Nothing we enforce but is used in a couple of places

```suggestion
		:class="{ draggedOver, 'editor--mobile': isMobile }"
```

Reply:
To then also adjust on the css part